java/de/yessoft/android/mohammad/services/MessageReceiver.java

```diff
@@ -0,0 +1,60 @@
+package de.yessoft.android.mohammad.services;
+
+import android.app.Notification;
+import android.app.NotificationManager;
+import android.app.PendingIntent;
+import android.content.Intent;
+import android.media.RingtoneManager;
+import android.net.Uri;
+
+import com.google.firebase.messaging.FirebaseMessagingService;
+import com.google.firebase.messaging.RemoteMessage;
+
+import de.yessoft.android.mohammad.R;
+import de.yessoft.android.mohammad.ui.MainActivity.MainActivity;
+
+public class MessageReceiver extends FirebaseMessagingService {
+    // private static final String TAG = "ML_MessageReceiver";
```

Comment:
please removed the unused code

Reply:
This is used for debug purposes, and i keep it as a hint for other contributers on how i handle tags in order to get the same development debug methodology, But it should be removed on the production level, i can see the point